app/db/session.py

```
@@ -0,0 +1,13 @@
+from sqlalchemy.ext.asyncio import AsyncSession, async_sessionmaker
```

Ruff (0.11.9)

`sqlalchemy.ext.asyncio.AsyncSession` imported but unused

Remove unused import: `sqlalchemy.ext.asyncio.AsyncSession`

(F401)